LOTUS_RELEASE_FLOW.md

```diff
@@ -95,12 +95,29 @@ By policy, the team will usually wait until about 3 weeks after the final releas
 
 Unless a security issue is actively being exploited or a significant number of users are unable to update to the latest version, security fixes will not be backported to previous releases.
 
+## Branch and Tag Strategy
```

Comment:
Hmm, in re-reading https://github.com/filecoin-project/lotus/pull/12322, I'm realizing there is a lot of overlap with the "Release Process" section.  I think we could merge them.  

Maybe the heading becomes "Release Process Including Branch and Tag Strategy"?

Or we could jsut do

```
## Release Process
See https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md
```

Reply:
> Or we could just do
> ```
> ## Release Process
> See https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md
> ```

I like this approach. Will incorporate it in https://github.com/filecoin-project/lotus/pull/12322 once this has merged into the `phi/update-release-flow` branch - to not cause any conflicts that needs to be resolved in this PR: